Python/Product/Analysis/Interpreter/Ast/AstPythonInterpreter.cs

```diff
@@ -98,7 +99,7 @@ class AstPythonInterpreter : IPythonInterpreter, IModuleContext, ICanFindModuleM
             return res;
         }
 
-        private async Task<IReadOnlyDictionary<string, string>> GetUserSearchPathPackagesAsync() {
+        private async Task<IReadOnlyDictionary<string, string>> GetUserSearchPathPackagesAsync(CancellationToken cancel) {
```

Comment:
I'd prefer to have full name for the parameter: `CancellationToken cancellationToken`